SA/design_concept/content/design/functional_requirements.typ

```diff
@@ -37,38 +37,42 @@ By default, "user" in the use case description refers to the "student user".
     caption: "Use Case Diagram"
 )<use_case_diagram>
 
+As the aim of this project is to find a visual representation of functional programming,
+the use case descriptions are kept very brief and only state the intention behind the use case but not how it must be achieved.
+
 === UC1 - Simple Function Composition
-A user wants to compose a simple function using pre-defined functions, e.g., Integer parameters.
+A user wants to execute their visually composed functions to see the effects of their functions on data.
```

Comment:
Is this the right UC? Or did something get mixed up?

Reply:
Moved to the right place